portal-test/src/com/liferay/portal/kernel/service/test/ServiceTestUtil.java

```diff
@@ -86,232 +32,7 @@
-				DestinationNames.DOCUMENT_LIBRARY_PDF_PROCESSOR);
-			Filter rawMetaDataProcessorFilter = _registerDestinationFilter(
-				DestinationNames.DOCUMENT_LIBRARY_RAW_METADATA_PROCESSOR);
-			Filter videoProcessorFilter = _registerDestinationFilter(
-				DestinationNames.DOCUMENT_LIBRARY_VIDEO_PROCESSOR);
-
-			serviceDependencyManager.registerDependencies(
-				audioProcessorFilter, imageProcessFilter, pdfProcessorFilter,
-				rawMetaDataProcessorFilter, videoProcessorFilter);
-
-			serviceDependencyManager.waitForDependencies();
-
-			_replaceWithSynchronousDestination(
-				DestinationNames.DOCUMENT_LIBRARY_AUDIO_PROCESSOR);
-			_replaceWithSynchronousDestination(
-				DestinationNames.DOCUMENT_LIBRARY_IMAGE_PROCESSOR);
-			_replaceWithSynchronousDestination(
-				DestinationNames.DOCUMENT_LIBRARY_PDF_PROCESSOR);
-			_replaceWithSynchronousDestination(
-				DestinationNames.DOCUMENT_LIBRARY_RAW_METADATA_PROCESSOR);
-			_replaceWithSynchronousDestination(
-				DestinationNames.DOCUMENT_LIBRARY_VIDEO_PROCESSOR);
-		}
-
-		// Shutdown
-
-		Registry registry = RegistryUtil.getRegistry();
-
-		HashMap<String, Object> messageBusProperties =
-			HashMapBuilder.<String, Object>put(
-				"service.ranking", Integer.MAX_VALUE
-			).build();
-
-		registry.registerService(
-			MessageBus.class, _messageBusWrapper, messageBusProperties);
-
-		if (MessageBusUtil.getMessageBus() != _messageBusWrapper) {
-			throw new IllegalStateException("MessageBus should be set");
-		}
-
-		HashMap<String, Object> portalExecutorManagerProperties =
-			HashMapBuilder.<String, Object>put(
-				"service.ranking", Integer.MAX_VALUE
-			).build();
-
-		registry.registerService(
-			PortalExecutorManager.class, _portalExecutorManagerWrapper,
-			portalExecutorManagerProperties);
-
-		// Class names
-
-		_checkClassNames();
-
-		// Resource actions
-
-		try {
-			_checkResourceActions();
-		}
-		catch (Exception e) {
-			_log.error(e, e);
-		}
-
-		// Workflow
-
-		PortalRegisterTestUtil.registerWorkflowHandlers();
-
-		// Company
-
-		if (!ArrayUtil.contains(
-				PortalInstances.getWebIds(), TestPropsValues.COMPANY_WEB_ID)) {
-
-			try {
-				CompanyLocalServiceUtil.checkCompany(
-					TestPropsValues.COMPANY_WEB_ID);
-			}
-			catch (Exception e) {
-				_log.error(e, e);
-			}
-		}
-	}
-
-	/**
-	 * @deprecated As of Mueller (7.2.x), with no direct replacement
-	 */
-	@Deprecated
-	public static Date newDate() throws Exception {
-		return new Date();
-	}
-
-	/**
-	 * @deprecated As of Mueller (7.2.x), with no direct replacement
-	 */
-	@Deprecated
-	public static Date newDate(int month, int day, int year) throws Exception {
-		Calendar calendar = new GregorianCalendar();
-
-		calendar.set(Calendar.MONTH, month);
-		calendar.set(Calendar.DATE, day);
-		calendar.set(Calendar.YEAR, year);
-
-		return calendar.getTime();
-	}
-
-	public static void setUser(User user) throws Exception {
+	public static void setUser(User user) {
```

Comment:
@dantewang @ChiZelin Please merge this class into `com.liferay.portal.kernel.test.util.UserTestUtil`. There is only one method left and it is related to user.

Tina.